cmd/airplane/tasks/dev/dev.go

```diff
@@ -72,7 +73,33 @@ func New(c *cli.Config) *cobra.Command {
 				} else if len(args) > 1 {
 					return errors.New("Multiple dev server roots detected, please supply only one directory to discover tasks and views")
 				} else {
+					// Use absolute path to dev root to allow the local dev server to more easily calculate relative paths.
 					cfg.fileOrDir = args[0]
+					if cfg.fileOrDir, err = filepath.Abs(cfg.fileOrDir); err != nil {
+						return err
+					}
+				}
+
+				if cfg.devConfigPath == "" {
+					var devDir string
+					// Calling filepath.Dir on a directory path that doesn't end in '/' returns that directory's parent -
+					// we want to return the directory itself in that case, and so we check if cfg.fileOrDir is a directory.
+					if info, err := os.Stat(cfg.fileOrDir); err != nil {
+						return err
+					} else if info.IsDir() {
+						devDir = cfg.fileOrDir
+					} else {
+						devDir = filepath.Dir(cfg.fileOrDir)
+					}
+
+					// Recursively search for dev config file from the dev dir until root.
```

Comment:
Root is confusing (user root vs. task root)

```suggestion
					// Recursively search for dev config file, starting from the dev dir.
```